extensions/wikia/Recirculation/WikiRecommendations.class.php

```diff
@@ -619,6 +620,21 @@ public static function getRecommendations( $contentLanguage ) {
 		return $recommendations;
 	}
 
+	private static function fallbackToSupportedLanguages( $language ) {
+		switch ( $language ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a "case default" clause to this "switch" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)